backend/persistent/QueryScheduler.js

```diff
@@ -387,19 +387,25 @@ class QueryScheduler {
                 */
                 throw new Error(`MetadataError: ${e.message}`);
             });
-        }).then(() => {
+        }).then(res => {
             Logger.log(`Request to Plotly for creating a grid took ${process.hrtime(startTime)[0]} seconds`, 2);
             Logger.log(`Grid ${fid} has been updated.`, 2);
 
+            if (res.status !== 200) {
```

Comment:
I think the PR was OK without this change. The error is already thrown [here](https://github.com/plotly/falcon-sql-client/blob/7a7483790488f897362479c6280658ab24ee20e5/backend/persistent/QueryScheduler.js#L361-L364).

---

Note that `res` here is `undefined` when the execution goes through https://github.com/plotly/falcon-sql-client/blob/7a7483790488f897362479c6280658ab24ee20e5/backend/persistent/QueryScheduler.js#L338 or https://github.com/plotly/falcon-sql-client/blob/7a7483790488f897362479c6280658ab24ee20e5/backend/persistent/QueryScheduler.js#L359

Reply:
@n-riesco without that change, wouldn't the error be swallowed if `patchGrid` responds non-200, but does not throw?